llvm/lib/Target/X86/X86.td

```diff
@@ -1249,6 +1249,18 @@ def ProcessorFeatures {
   list<SubtargetFeature> ARLSFeatures =
     !listconcat(SRFFeatures, ARLSAdditionalFeatures);
 
+  // Pantherlake
+  list<SubtargetFeature> PTLAdditionalFeatures = [FeaturePREFETCHI];
+  list<SubtargetFeature> PTLFeatures =
+    !listconcat(ARLSFeatures, PTLAdditionalFeatures);
```

Comment:
EAS doesn't say `Panther Lake` has these `ARLSAdditionalFeatures`

Reply:
I referred to xed patch: https://github.com/intelxed/xed/blob/main/datafiles/ptl/ptl-chips.txt